src/resizing.jl

```diff
@@ -182,16 +191,43 @@ odims(original, i, short_size) = oftype(first(short_size), axes(original, i))
 
 """
     imresize(img, sz) -> imgr
-    imresize(img, inds) -> imgr
+    imresize(img, ratio::Real) -> imgr
+    imresize(img, inds::Integer...) -> imgr
+
+Change `img` to be of size `sz` (or to have indices `inds`). If `ratio` is used, then
+`sz = ceil(Int, size(img).*ratio)`. This interpolates the values at sub-pixel locations.
+If you are shrinking the image, you risk aliasing unless you low-pass filter `img` first.
+
+# Example:
+```julia
+σ = map((o,n)->0.75*o/n, size(img), sz)
+kern = KernelFactors.gaussian(σ)   # from ImageFiltering
+imgr = imresize(imfilter(img, kern, NA()), sz)
+```
+
+# Example:
+```julia
+julia> img = testimage("lena_gray_256")
+julia> imresize(img, 128, 128) # 128*128
+julia> imresize(img, 1:128, 1:128) # 128*128
+julia> imresize(img, [128,128]) # 128*128
+julia> imresize(img, [1:128,1:128]) # 128*128
+julia> imresize(img, (128,128)) # 128*128
+julia> imresize(img, (1:128,1:128)) # 128*128
+julia> imresize(img, (1:128,)) # 128*256
+julia> imresize(img, 128) # 128*256
+julia> imresize(img, 0.5) # 128*128
+```
+
+!!! info
 
-Change `img` to be of size `sz` (or to have indices `inds`). This
-interpolates the values at sub-pixel locations. If you are shrinking
-the image, you risk aliasing unless you low-pass filter `img`
-first. For example:
+- for N-dimensional image/signal, `imresize(img, arg)` treats `arg::Real` as `ratio`, and `arg::Integer` as `sz`. In the latter case, only the first dimension will be resized.
 
-    σ = map((o,n)->0.75*o/n, size(img), sz)
-    kern = KernelFactors.gaussian(σ)   # from ImageFiltering
-    imgr = imresize(imfilter(img, kern, NA()), sz)
+```julia
+julia> img_1d = 0:0.1:1 .|> n0f8
+julia> imresize(img_1d, 3) # 3*10-element Array{N0f8,1}
+julia> imresize(img_1d, 3.0) # 30*30-element Array{N0f8,1}
+```
 
```

Comment:
I think doing this would not introduce ambiguity. For `Integer` it means size, and for other `Real` type it means ratio.

Reply:
the info block is not indented properly. The body of the block should all be indented 4 spaces to be aligned with the ford `info` in `!!! info`. see https://docs.julialang.org/en/v1/manual/documentation/#Admonitions-1